src/ACESVGjsPanel.tsx

```diff
@@ -123,38 +132,41 @@ export class ACESVGPanel extends PureComponent<Props, PanelState> {
       }
       currentNode = svgWalker.nextNode() as Element;
     }
-    this.setState({ svgMappings: [...svgMappings], initialized: false });
-    this.props.options.svgMappings = [...svgMappings];
```

Comment:
mutating props directly is asking for trouble...

Reply:
Yeah, I only resorted back to that when I couldn't get anything else to work. :/